README.md

```diff
@@ -61,6 +61,10 @@ Please see the Examples section for more details.
 
 ## Versions
 
+### 2.3.1
```

Comment:
Per [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module), don't change the version of the module.  Document new changes under an unreleased version.